resources/jmqttd/jeedom/jeedom.py

```diff
@@ -157,7 +157,7 @@ def convert_log_level(level = 'error'):
 
 	@staticmethod
 	def set_log_level(level = 'error'):
-		FORMAT = '[%(asctime)-15s][%(levelname)s] : %(message)s'
+		FORMAT = '[%(asctime)-15s][%(levelname) 7s] : %(message)s'
```

Comment:
lot of examples use this format for log level : [%(levelname)-8s]
Isn't it better than [%(levelname) 7s]
ref : https://docs.python.org/3/howto/logging-cookbook.html

Reply:
All right done !